heartbeat/tests/system/test_autodiscovery.py

```diff
@@ -0,0 +1,61 @@
+import os
+from heartbeat import BaseTest
+import unittest
+import re
+
+from beat.beat import INTEGRATION_TESTS
+
+
+class TestAutodiscover(BaseTest):
+    """
+    Test heartbeat autodiscover
+    """
+    @unittest.skipIf(not INTEGRATION_TESTS or
+                     os.getenv("TESTING_ENVIRONMENT") == "2x",
+                     "integration test not available on 2.x")
+    def test_docker(self):
+        """
+        Test docker autodiscover starts modules from templates
+        """
+        import docker
+        docker_client = docker.from_env()
+
+        self.render_config_template(
```

Comment:
For later: As we have ES running anyways, we could add some additional tests for http.

Reply:
Discussed this on slack with @ruflin , agreed that adding a second test was not appropriate due to the fact that we're testing autodiscovery, not reloading.